kolibri/plugins/media_player/assets/src/views/MediaPlayerTranscript/TranscriptCue.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+
+  <a
+    role="button"
+    tabindex="0"
+    :class="[
+      'transcript-cue',
+      { active },
+      $computedClass(style)
+    ]"
```

Comment:
one thing I learned recently is that you can include both `class` and `:class` in the same element, e.g.

```
    class="transcript-cue"
    :class="[
      { active },
      $computedClass(style)
    ]"
```

I don't think that's an improvement in this case, but just FYI in case that's new to you too